### DISCUSSION TITLE
PRs in stale status

### DISCUSSION THREAD

**Discussion by quequiel** [netbox-community/devicetype-library#967] [Q&A] [answered]
Hi everyone!
This is my first time committing so I wanted to make sure I wasn´t missing any step. I made a copule of PRs (#935 and #937 ) but now they appear as stale. If I understood correctly, some maintainer must approve the workflow as I am a first time contributor but as that didn´t happen the requests have been marked as stale. Have I missed any step? Are new contributors being accepted?

Alejandro

**Comment by madecto** [comment]
Hello @quequiel,
just my 2 cents here, but it appears all of our PRs are frozen until one of the maintainers review them: if you look on the GitHub Insights page for this repo you'll discover no PR has been approved since mid october: I guess everybody's overworked.

**Reply by quequiel** [reply]
Hi @madecto,
yes, I assumed that. Just wanted to validate my assumption was correct and I wasn't missing anything on my side.

**Comment by danner26** [comment, marked as answer]
Hey @quequiel @madecto, the team has been very busy getting the latest release ready. I am here to alleviate some of that pressure, and will be reviewing/merging the PRs for them
If you need anything please feel free to reach out

**Reply by madecto** [reply]
Thanks @danner26, much appreciated and I refer both to the project & the involved people's efforts.
